build.gradle

```diff
@@ -23,8 +23,12 @@ buildscript {
 }
 
 allprojects {
+    tasks.withType(JavaCompile).configureEach {
+        options.fork = true
+    }
+
     repositories {
         google()
         jcenter()
     }
-}
+}
```

Comment:
Please do not remove the newline at EOL.

Reply:
We have modified this file